### DISCUSSION TITLE
Support for SSAS/AAS

### DISCUSSION THREAD

**Discussion by josef78** [sql-bi/Bravo#515] [Ideas]
Please, add support to connect to tabular models in SQL Server Analysis Services and Azure Analysis Services...

**Comment by albertospelta** [comment]
Hi @josef78, you can connect to an SSAS instance (SQL Server Analysis Services or Azure Analysis Services) by launching Bravo via the Power BI External Tools button from a Power BI Desktop report already connected to that SSAS instance.

**Reply by josef78** [reply]
Good, it's working from classic "Power BI Desktop" including unused columns. It's nice. Thank you so much.

But it not working from "Power BI Desktop RS" which more common in enterprises with on-premises enviroments (SSAS + PBI RS + PBI Desktop RS). Also it's not working when I try connect from Bravo to Power BI Desktop RS which is live connected to SSAS.

I think it would be very nice to be able to connect to PBI Desktop RS live connected to SSAS, or directly connect to SSAS/AAS from Bravo.

**Reply by albertospelta** [reply]
For a direct connection to SSAS/AAS from Bravo we have something like this already on the backlog (see https://github.com/sql-bi/Bravo/issues/254) - you can follow the issue to stay updated on the release of this feature.

Regarding "Power BI Desktop RS," this is an edition of Power BI Desktop that is currently untested. 
I am going to open a new issue to keep track of this as a feature enhancement. Thanks for this suggestion @josef78 !

**Reply by albertospelta** [reply]
FYI 
_Add support for Power BI Desktop RS_  https://github.com/sql-bi/Bravo/issues/518
